### PR TITLE
Add opensearch tests

### PR DESCRIPTION
**Pull request by jelly**
Add additional tests for our opensearch endpoints

### CODE REVIEW

**Comment by angvp** [comment]
@jelly I'd keep the naming scheme pythonic test_packages and test_packages_suggest for the functions instead camel case names, also, not sure if the test runner takes these?

File: packages/tests.py [hunk 1]
```diff
@@ -163,4 +163,21 @@ def test_filter_packager_unknown(self):
         self.assertEqual(response.status_code, 200)
         self.assertIn('5 matching packages found', response.content)
 
+class OpenSearch(TestCase):
+    fixtures = ['main/fixtures/arches.json', 'main/fixtures/repos.json',
+                'main/fixtures/package.json']
+
+
+    def testPackages(self):
```
Comment (angvp) [inline packages/tests.py]: test_packages instead

File: packages/tests.py [hunk 2]
```diff
@@ -163,4 +163,21 @@ def test_filter_packager_unknown(self):
...
+        response = self.client.get('/opensearch/packages/')
+        self.assertEqual(response.status_code, 200)
+
+    def testPackagesSuggest(self):
```
Comment (angvp) [inline packages/tests.py]: test_packages_suggest

File: packages/tests.py (hunk 1 above)
Comment (jelly) [reply packages/tests.py]: Fixed

File: packages/tests.py (hunk 2 above)
Comment (jelly) [reply packages/tests.py]: fixed

**Pull request merged**